commit 6135c97d7b02a4506fcb86f9a8e979230e53087f
Author: Tim Serong <tserong@suse.com>
Date:   Fri May 19 22:10:03 2017 +1000

    mgr: add generic --help output
    
    Creating an MgrStandby in order to call its usage() method has
    two problems:
    
    1) MgrStandby wants to connect to a running cluster; if the
       cluster isn't up, you'll be left staring at a blinking cursor.
    2) MgrStandby::usage() doesn't print anything anyway.
    
    This commit takes pretty much the same approach to printing
    usage information as ceph_mon.cc, ceph_osd.cc and ceph_mds.cc.
    
    Signed-off-by: Tim Serong <tserong@suse.com>

diff --git a/src/mgr/MgrStandby.h b/src/mgr/MgrStandby.h
index d33c633dc42..9d160a10642 100644
--- a/src/mgr/MgrStandby.h
+++ b/src/mgr/MgrStandby.h
@@ -73,7 +73,6 @@ public:
 
   int init();
   void shutdown();
-  void usage() {}
   int main(vector<const char *> args);
   void handle_signal(int signum);
   void send_beacon();
